app/assets/javascripts/initializers/initializeRegistration.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
```

Comment:
Refactor (blocking): `initializeRegistration` will get loaded on every page, since `//= require_tree ./initializers` loads all files in the `initializers` folder in base.js.erb.

Can we load this only for the registration page? If that's not possible, consider adding it to packs/base.js and dynamically import it based on whether admin registration is required.

Importing it into packs/base.js will also allow us to use ES module. Currently, all this JavaScript is scoped globally.

Reply:
@nickytonline thanks for this feedback! Makes a lot of sense - I'll look into this (hopefully today 😊 )